grader_support/gradelib.py

```diff
@@ -238,11 +239,7 @@ def _tokens(code):
     A wrapper around tokenize.generate_tokens.
     """
     # Protect against pathological inputs: http://bugs.python.org/issue16152
```

Comment:
This comment should also be removed, as the bug is fixed and the bug workaround is removed.